CONTRIBUTING.md

```diff
@@ -0,0 +1,192 @@
+details and the rationale for the proposed change.
+
+## Pull requests
+
+If you’d like to propose and collaborate on changes, open a [pull request][]!
+
+Here are a few things you can do that will increase the likelihood of having
+your pull request merged:
+
+- Follow standards for style and code quality.
+- Write tests when applicable.
+- Keep your change as focused as possible. If there are multiple changes you
+  would like to make that are not dependent upon each other, consider submitting
+  them as separate pull requests.
+- Write a [good commit message][].
+
+Contributions to this project are [released][contrib-license] to the public
+under the project’s open source license. The license for a project is located in
+a file named `LICENSE` in the root directory of the repository.
+
+## Project documentation
+
+The `README.md` in the root of the repository should contain or link to project
+documentation. If you cannot find the documentation you’re looking for, please
+file a GitHub issue with details of what you’d like to see documented.
+
+## Questions or thoughts?
+
+Support requests (e.g., asking questions) or feedback (e.g., constructive
+criticism) may be directed to the [`@openinf`][twitter-account] support account
+on Twitter.
+
+## Additional resources
+
+- [How to Contribute to Open Source](https://opensource.guide/how-to-contribute/)
+- [Using Pull Requests](https://help.github.com/articles/about-pull-requests/)
+- [GitHub Help](https://help.github.com)
+
+## Code of Conduct
+
+By participating in this project, you are expected to uphold our [Code of
+Conduct][code_of_conduct].
+
+## Developer’s Certificate of Origin 1.1
+
+By making a contribution to this project, I certify that:
+
+- (a) The contribution was created in whole or in part by me and I have the
+  right to submit it under the open source license indicated in the file; or
+
+- (b) The contribution is based upon previous work that, to the best of my
+  knowledge, is covered under an appropriate open source license and I have the
+  right under that license to submit that work with modifications, whether
+  created in whole or in part by me, under the same open source license (unless
+  I am permitted to submit under a different license), as indicated in the file;
+  or
+
+- (c) The contribution was provided directly to me by some other person who
+  certified (a), (b) or (c) and I have not modified it.
+
+- (d) I understand and agree that this project and the contribution are public
+  and that a record of the contribution (including all personal information I
+  submit with it, including my sign-off) is maintained indefinitely and may be
+  redistributed consistent with this project or the open source license(s)
+  involved.
+
+## Reporting Security Issues
+
+Please see the [`SECURITY.md`] file.
+
+## Attribution
+
+[Developer Certificate of Origin Legal Text](https://developercertificate.org/)
```

Comment:
```suggestion
```

<br />

I do not think this should be an h2-level heading as it is here and should instead use the 
GitHub-flavored Markdown for footnotes as attribution[^1] if possible&hellip;

[^1]: https://github.blog/changelog/2021-09-30-footnotes-now-supported-in-markdown-fields/